test/Makefile

```diff
@@ -18,10 +18,13 @@ all: test
 
 NON_INTERACTIVE_FLAG=--skip-resource=false --skip-controller=false --skip-admission-controller=false
 
-test: build
+test: build check
 	go test ./pkg/...
 	bash -c "find pkg/apis/ -name apiserver.local.config | xargs rm -rf"
 
+check: build
+	go vet $$(go list ./... | grep -vE "_generated" | grep -vE "plugin/admission")
```

Comment:
Some go files under the `plugin/admission` cannot pass go vet check, workaround it for now.
```
go vet $(go list ./... | grep -vE "_generated")
# sigs.k8s.io/apiserver-builder-alpha/test/plugin/admission/snapshot
plugin/admission/snapshot/admission.go:54:58: undefined: "sigs.k8s.io/apiserver-builder-alpha/test/vendor/k8s.io/apiserver/pkg/admission".ObjectInterface
plugin/admission/snapshot/admission.go:58:61: undefined: "sigs.k8s.io/apiserver-builder-alpha/test/vendor/k8s.io/apiserver/pkg/admission".ObjectInterface
# sigs.k8s.io/apiserver-builder-alpha/test/plugin/admission/snapshotclaim
plugin/admission/snapshotclaim/admission.go:54:63: undefined: "sigs.k8s.io/apiserver-builder-alpha/test/vendor/k8s.io/apiserver/pkg/admission".ObjectInterface
plugin/admission/snapshotclaim/admission.go:58:66: undefined: "sigs.k8s.io/apiserver-builder-alpha/test/vendor/k8s.io/apiserver/pkg/admission".ObjectInterface
# sigs.k8s.io/apiserver-builder-alpha/test/plugin/admission/volume
plugin/admission/volume/admission.go:54:56: undefined: "sigs.k8s.io/apiserver-builder-alpha/test/vendor/k8s.io/apiserver/pkg/admission".ObjectInterface
plugin/admission/volume/admission.go:58:59: undefined: "sigs.k8s.io/apiserver-builder-alpha/test/vendor/k8s.io/apiserver/pkg/admission".ObjectInterface
# sigs.k8s.io/apiserver-builder-alpha/test/plugin/admission/volumeclaim
plugin/admission/volumeclaim/admission.go:54:61: undefined: "sigs.k8s.io/apiserver-builder-alpha/test/vendor/k8s.io/apiserver/pkg/admission".ObjectInterface
plugin/admission/volumeclaim/admission.go:58:64: undefined: "sigs.k8s.io/apiserver-builder-alpha/test/vendor/k8s.io/apiserver/pkg/admission".ObjectInterface
```
Looks like that these cannot even compile, the test succeed formerly because the builded binaries does not involve these go files.

Reply:
I believe it is introduced by the dependency bump, now that we have upgraded the kubernetes dependencies to `1.16`, apiserver-boot should also be updated: https://github.com/kubernetes-sigs/apiserver-builder-alpha/blob/master/cmd/apiserver-boot/boot/create/resource.go#L509

 Tracked by: https://github.com/kubernetes-sigs/apiserver-builder-alpha/issues/437

Reply:
thanks, i fixed compile issue by https://github.com/kubernetes-sigs/apiserver-builder-alpha/pull/438. rebase onto that and vetting admission packages should work now